source/code/include/playfab/PlayFabEventPipeline.h

```diff
@@ -95,10 +102,13 @@ namespace PlayFab
         // that would allow to quickly map a pointer (like void* customData) to a batch (like a std::vector<std::shared_ptr<const IPlayFabEmitEventRequest>>).
         std::mutex inFlightMutex;
         std::unordered_map<void*, std::vector<std::shared_ptr<const IPlayFabEmitEventRequest>>> batchesInFlight;
+        //std::vector<std::shared_ptr<const IPlayFabEmitEventRequest>> batch;
```

Comment:
we should probably remove this.

Reply:
We were able to remove batch as a member to help potential threading issues on our codebase by making it more narrowly scoped (as a parameter in SendBatch, which in Scott's review was empty)

Reply:
I guess I just meant we should remove this commented out line. :)